proto/cosmos/group/v1/query.proto

```diff
@@ -85,7 +85,7 @@ service Query {
   };
 
   // Groups queries all groups in state.
-  // 
+  //
```

Comment:
this seems uneeded

Reply:
I think is something weird git did i did not touch this file